adapter/PreciceInterface.c

```diff
@@ -469,17 +467,26 @@ void PreciceInterface_ConfigureCouplingData( PreciceInterface * interface, Simul
 			interface->kDeltaReadDataID = precicec_getDataID( config->readDataNames[i], interface->faceCentersMeshID );
 			printf( "Read data '%s' found.\n", config->readDataNames[i] );
 		}
-        	else if ( strcmp1( config->readDataNames[i], "Forces" + i ) == 0 )
-        	{
-            		PreciceInterface_EnsureValidNodesMeshID( interface );
-            		interface->readData = FORCES;
-            		interface->xforcIndices = malloc( interface->numNodes * 3 * sizeof( int ) );
-            		interface->forcesDataID = precicec_getDataID( config->readDataNames[i], interface->nodesMeshID );
-            		getXforcIndices( interface->nodeIDs, interface->numNodes, sim->nforc, sim->ikforc, sim->ilforc, interface->xforcIndices );
-            		printf( "Read data '%s' found.\n", config->readDataNames[i] );
-            		break;
-            
-        	}
+        else if ( strcmp1( config->readDataNames[i], "Forces" + i ) == 0 )
+        {
+            PreciceInterface_EnsureValidNodesMeshID( interface );
+            interface->readData = FORCES;
+            interface->xforcIndices = malloc( interface->numNodes * 3 * sizeof( int ) );
+            interface->forcesDataID = precicec_getDataID( config->readDataNames[i], interface->nodesMeshID );
+            getXforcIndices( interface->nodeIDs, interface->numNodes, sim->nforc, sim->ikforc, sim->ilforc, interface->xforcIndices );
+            printf( "Read data '%s' found.\n", config->readDataNames[i] );
+            break;
+        }
+        else if ( strcmp1( config->readDataNames[i], "Displacements" + i ) == 0 )
+        {
+        	PreciceInterface_EnsureValidNodesMeshID( interface );
+        	interface->readData = DISPLACEMENTS;
+			interface->xbounIndices = malloc( interface->numNodes * 3 * sizeof( int ) );
+            interface->displacementsDataID = precicec_getDataID( config->readDataNames[i], interface->nodesMeshID );
+			getXbounIndices( interface->nodeIDs, interface->numNodes, sim->nboun, sim->ikboun, sim->ilboun, interface->xbounIndices, DISPLACEMENTS );
```

Comment:
White spaces seem to have been mixed up a little. Please correct this (at several positions in the code).